frontend/src/js/components/viewer/DocumentMetadata.js

```diff
@@ -104,7 +104,7 @@ export class DocumentMetadata extends React.Component {
                 <div className='sidebar__title'>
                     Actions
                 </div>
-                <ViewerActions resource={this.props.resource} config={this.props.config} />
+                <ViewerActions resource={this.props.resource} config={this.props.config} isAdmin={this.props.myPermissions.includes('CanPerformAdminOperations')} />
```

Comment:
We can check if there's child elements using the state we have in the redux store to hide this button if there are subresources.

Reply:
This is handled by https://github.com/guardian/giant/blob/pm-delete/frontend/src/js/components/viewer/DeleteButtonModal.tsx#L15